react/features/toolbox/components/web/Toolbox.js

```diff
@@ -553,6 +560,21 @@ class Toolbox extends Component<Props, State> {
 
         this._doToggleChat();
     }
+	
+	_onShortcutToggleVideoQuality: () => void;
+
+    /**
+     * Creates an analytics keyboard shortcut event and dispatches an action for
+     * toggling the display of Video Quality.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onShortcutToggleVideoQuality() {
+        sendAnalytics(createShortcutEvent('Video.quality'));
```

Comment:
Other event names passed into createShortcutEvent don't follow this casing pattern.